File: engine/model/src/main/scala/tools/packet/parcel/RemoveDueyItem.scala
package tools.packet.parcel

import net.opcodes.SendOpcode
import tools.packet.PacketInput

class RemoveDueyItem(private var _remove: Boolean, private var _packageId: Int) extends PacketInput {
  def remove: Boolean = _remove

  def packageId: Int = _packageId

  override def opcode(): SendOpcode = SendOpcode.PARCEL
}

File: engine/engine-model/src/main/java/tools/packet/parcel/RemoveDueyItem.java
package tools.packet.parcel;

import net.opcodes.SendOpcode;
import tools.packet.PacketInput;

public record RemoveDueyItem(Boolean remove, Integer packageId) implements PacketInput {
   @Override
   public SendOpcode opcode() {
      return SendOpcode.PARCEL;
   }
}